docs: new guide for outgoing payment grant for future payments (#607)
interledger/open-payments: docs/src/content/docs/guides/outgoing-grant-no-recipients.mdx

---

> title: Get an outgoing payment grant without recipients

Reviewer: ```suggestion
title: Get an outgoing payment grant for future payments
```
Because technically all of our the outgoing payment grant requests that we have documented do not correspond to a recipient

      start => ["redirect"],
      finish => [
        method => "redirect",
        uri => "https://cloudninebank.example.com/finish/T8jw5Xy",
        nonce => "NONCE",
      ],
    ],
  ]
);
```
</TabItem>
</Tabs>

#### About the interval

The interval used in this guide is `R3/2025-05-20T13:00:00Z/P1M`. Remember that your user wants to send payments up to $100 CAD a month for three months. The interval breaks down like this:

- `R[3]/` is the number of repetitions - three.
- `2025-05-20` is the start date of the repeating interval - 20 May 2025
- `T13:00:00Z/` is the start time of the repeating interval - 1:00 PM UTC.
- `[P1M]` is the period of time between each interval - one month. Used with `R[3]`, you have a grant that’s valid once a month for three months.

Altogether, your user can send up to $100 CAD from:

- 1 PM UTC on 20 May 2025 through 12:59 PM UCT on 20 June 2025
- 1 PM UTC on 20 June 2025 through 12:59 PM UTC on 20 July 2025
- 1 PM UTC on 20 July 2025 through 12:59 PM UCT on 20 August 2025

### 4. Start interaction with the user

<StartInteraction />

### 5. Finish interaction with the user

<FinishInteraction />

### 6. Request a grant continuation

> Use the interaction reference (`interact_ref`) returned in the previous step to continue the grant request.

Reviewer: nit: "Returned in the redirect URI's query parameters" to make it a bit more explicit
